dags.yaml

```diff
@@ -2161,11 +2161,12 @@ bqetl_alchemer:
     owner: ago@mozilla.com
     email:
       - ago@mozilla.com
+      - eshallal@mozilla.com
     email_on_failure: true
     email_on_retry: false
-    start_date: "2025-02-18"
+    start_date: "2023-06-01"
```

Comment:
Not sure if you still intend to backfill, but I think by default all our dags have `catchup=False` so it won't automatically run all the past dates for this dag.  If you want to backfill, [the backfill dag](https://workflow.telemetry.mozilla.org/dags/backfill/grid) might work.  Alternatively, you could add support for the catchup property to the bigquery-etl dag generation